cherrypy/_cpcompat.py

```diff
@@ -332,3 +332,13 @@ def _args_from_interpreter_flags():
             args.append('-W' + opt)
 
         return args
+
+if six.PY2:
```

Comment:
I'd use try/except ImportError approach. `html.escape()` appeared in 3.2, so it will fail for 3.0-3.1.